samples/dashboards/components/climate-all/demo.js

```diff
@@ -665,44 +665,42 @@ async function setupBoard() {
                 highlight: true
             },
             dataGridOptions: {
-                cellHeight: 38,
-                editable: false,
                 columns: {
                     time: {
-                        show: false
+                        enabled: false
                     },
                     FD: {
-                        headerFormat: 'Days with Frost'
+                        headFormat: 'Days with Frost'
                     },
                     ID: {
-                        headerFormat: 'Days with Ice'
+                        headFormat: 'Days with Ice'
                     },
                     RR1: {
-                        headerFormat: 'Days with Rain'
+                        headFormat: 'Days with Rain'
                     },
                     TN: {
-                        show: false
+                        enabled: false
                     },
                     TX: {
-                        show: false
+                        enabled: false
                     },
                     TNC: {
-                        headerFormat: 'Average Temperature °C',
+                        headFormat: 'Average Temperature °C',
                         cellFormat: '{value:.1f}'
                     },
                     TNF: {
-                        headerFormat: 'Average Temperature °F',
+                        headFormat: 'Average Temperature °F',
                         cellFormat: '{value:.1f}',
-                        show: false
+                        enabled: false
                     },
                     TXC: {
-                        headerFormat: 'Maximal Temperature °C',
+                        headFormat: 'Maximal Temperature °C',
                         cellFormat: '{value:.1f}'
                     },
                     TXF: {
-                        headerFormat: 'Maximal Temperature °F',
+                        headFormat: 'Maximal Temperature °F',
```

Comment:
My first impression is: "headerFormat" sounds more intuitive in comparison with "headFormat".

Reply:
But it's a `head`, not a `header`.

Reply:
@karolkolodziej what do you think?

Reply:
We need to decide on a uniform nomenclature. If we decide on `header` instead of `head`, we need to change it everywhere.

Reply:
Technically we are not styling `<thead>` but its child `<th>` which stands for the header. So in this case maybe it makes more sense for `headerFormat`? But the whole structure should still be called `head`